.gitignore

```diff
@@ -27,6 +27,3 @@ Thumbs.db
 Carthage/
 
 Frameworks/
-
-.swiftlint-report.json
```

Comment:
It's true fastlane and reports can generate some files (in case the user runs these commands manually), we can ignore them or make the user remove them before committing.

Anyway, this name has changed and has to be removed.